### PR TITLE
update node version in Requirements

### PR DESCRIPTION
**Pull request by salus-sage**
related to issue #34

### CODE REVIEW

**Comment by BigBlueHat** [comment]
So, I don't think we want to bump the base requirements for the project until we're sure they're required for basic dependency building. It may be (as @tilgovi [pointed out](https://github.com/apache/incubator-annotator/issues/34#issuecomment-398467779)) that the development dependencies may be the ones with the higher requirements.

Regardless, I think there's some basic Continuous Integration (Travis or whoever) which would answer these basic questions for us (for the longer term).

**tl;dr** Let's not merge this just yet (and ideally without the strike through in any case).

**Comment by tilgovi** [comment]
Ah, I think I know what the right answer to this is.

We should set some recent 8.x version in the top-level package.json, because that's _just_ the yarn workspace package.json file and is not a published package. We should set 6.x in the engines field of every individual package, since that's what we target with the build.

And yes, we should get some CI set up soon.

**Comment by BigBlueHat** [comment]
Makes good sense. I've started with Travis over here (fwiw): https://travis-ci.org/BigBlueHat/incubator-annotator I'm open to other tooling, just started with what I had on hand.

I'll work that toward passing changing the reqs and such as needed.
